panels/user-accounts/cc-app-permissions.c

```diff
@@ -924,11 +926,25 @@ cc_app_permissions_class_init (CcAppPermissionsClass *klass)
 static void
 cc_app_permissions_init (CcAppPermissions *self)
 {
+  g_autoptr(GDBusConnection) system_bus = NULL;
+  g_autoptr(GError) error = NULL;
+
   gtk_widget_init_template (GTK_WIDGET (self));
 
   self->system_installation = flatpak_installation_new_system (NULL, NULL);
   self->user_installation = flatpak_installation_new_user (NULL, NULL);
 
+  self->cancellable = g_cancellable_new ();
+
+  system_bus = g_bus_get_sync (G_BUS_TYPE_SYSTEM, self->cancellable, &error);
```

Comment:
It’s not great to do synchronous IPC in the init function of a widget, but if you add a `FIXME` comment this will probably do for now, given that a lot of other things in g-c-c don’t do async properly already.

Reply:
Introduced a `FIXME`.
Thanks for the review.
Merging after jenkins-build.